build.sbt

```diff
@@ -2,85 +2,96 @@ import Dependencies._
 import xerial.sbt.Sonatype._
 
 lazy val qbeastCore = (project in file("core"))
-  .settings(name := "qbeast-core", libraryDependencies ++= Seq(apacheCommons % Test))
+  .settings(
+    name := "qbeast-core",
+    version := "0.1.0",
+    commonSettings,
+    releaseSettings,
+    libraryDependencies ++= Seq(apacheCommons % Test))
+
+lazy val qbeastSparkVersion = "0.2.0"
 
 // Projects
 lazy val qbeastSpark = (project in file("."))
   .enablePlugins(ScalaUnidocPlugin)
   .dependsOn(qbeastCore)
   .settings(
     name := "qbeast-spark",
+    version := qbeastSparkVersion,
+    commonSettings,
     libraryDependencies ++= Seq(
+      qbeastCoreDep,
+      deltaCore % Provided,
       sparkCore % Provided,
       sparkSql % Provided,
       hadoopClient % Provided,
-      deltaCore % Provided,
+      sparkFastTests % Test,
       amazonAws % Test,
       hadoopCommons % Test,
       hadoopAws % Test),
+    noWarningInConsole,
+    Compile / doc / scalacOptions ++= Seq(
+      "-doc-title",
+      "qbeast-spark",
+      "-doc-version",
+      qbeastSparkVersion,
+      "-doc-footer",
+      "Copyright 2022 Qbeast - Docs for version " + qbeastSparkVersion + " of qbeast-spark"),
     Test / parallelExecution := false,
     assembly / test := {},
     assembly / assemblyOption := (assembly / assemblyOption).value.copy(includeScala = false),
     publish / skip := true)
-  .settings(noWarningInConsole)
-
-qbeastSpark / Compile / doc / scalacOptions ++= Seq(
-  "-doc-title",
-  "qbeast-spark",
-  "-doc-version",
-  qbeast_spark_version,
-  "-doc-footer",
-  "Copyright 2022 Qbeast - Docs for version " + qbeast_spark_version + " of qbeast-spark")
 
 lazy val qbeastSparkNodep = (project in file("nodep"))
-  .settings(name := "qbeast-spark-nodep", Compile / packageBin := (qbeastSpark / assembly).value)
+  .settings(
```

Comment:
If we're keeping any of them, we might include the release settings inside the project's scope, instead of using ThisBuild:

`qbeastSparkNodep` would contain:
https://github.com/Qbeast-io/qbeast-spark/blob/04428ff8f035d6f9c365243ea6f96fe7feff1b22/build.sbt#L111-L121

and `qbeastSparkMaven`:
https://github.com/Qbeast-io/qbeast-spark/blob/04428ff8f035d6f9c365243ea6f96fe7feff1b22/build.sbt#L97-L109